app/core/functions.php

```diff
@@ -0,0 +1,77 @@
+<?php require 'database.php'; 
+
+/*
+*
+*
+* Global Functions 
+*
+*
+*/
+
+/* ###### INIT ###### */
+
+function init() {
+    // Default Page
+    $page = "search";
+    // get URL
+    $url = parseURL();
+    // update page if set
+    if(isset($url[0])){
+        if(file_exists('controllers/'. $url[0] . '.php')){
+            $page = $url[0];
+        }
+        else {
+            $page = '404';
+        }
+    }
+
+    // Require Controller
+    require_once 'controllers/' .$page. '.php';
```

Comment:
The way I've set up the MVC is the controller & the view are tied to the URL, so perhaps I move the function declarations out of the controllers and back in to functions.php as globals and then just call the function in the controller?